tox.ini

```diff
@@ -164,7 +164,7 @@ commands =
         examples/test/
 
 [testenv:pylint]
-basepython = python3
```

Comment:
Note:
The base-python and requirements file in a tox.ini
SHOULD be a consistent pair everywhere the
requirements is used.